universal-login-relayer/test/unit/MessageQueue.ts

```diff
@@ -40,7 +40,7 @@ describe('UNIT: Message Queue Service', async () => {
     await waitExpect(() => expect(executor.executeAndWait).to.be.calledOnce);
   });
 
-  it('should execute signedMessage if id added before start', async () => {
+  it('should execute signedMessage if it added before start', async () => {
```

Comment:
should execute pending signedMessage after `start`

Reply:
✔️